packages/devtools/src/commands/switchToFrame.ts

```diff
@@ -53,15 +62,15 @@ export default async function switchToFrame ({ id }) {
         /**
          * `page` has `frames` method while `frame` has `childFrames` method
          */
-        let getFrames = page.frames || page.childFrames
+        let getFrames = (page as unknown as Page).frames || page.childFrames
```

Comment:
Is ir possible to reuse Puppeteer types to avoid casting?

Reply:
we use Puppeteer types here already, the issue is that we use sometimes frames and sometimes pages which is a bug and should be cleaned up (as part of another PR).

Reply:
Are you going to work on it later on? Or we need an issue for tracking?

Reply:
I will create an issue for it.

Reply:
https://github.com/webdriverio/webdriverio/issues/6103